python/ray/util/client/server/proxier.py

```diff
@@ -144,7 +144,11 @@ def _get_unused_port(self) -> int:
             num_ports = len(self._free_ports)
             for _ in range(num_ports):
                 port = self._free_ports.pop(0)
-                s = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+                node_ip = ray.util.get_node_ip_address()
+                s = socket.socket(
+                    socket.AF_INET6 if is_ipv6(node_ip) else socket.AF_INET,
+                    socket.SOCK_STREAM,
+                )
```

Comment:
The port we get here is used to start a server that binds to `127.0.0.1` later so here we should always use socket.AF_INET. In a separate PR, we should change `127.0.0.1` to `get_localhost_ip()` and then here we should do `is_ipv6(get_localhost_ip())`. In this PR, let's just revert the change.